ksql-functional-tests/src/test/java/io/confluent/ksql/test/tools/KsqlTestingToolTest.java

```diff
@@ -189,7 +189,8 @@ public void shouldPropegateInsertValuesExecutorError() throws Exception {
 
     // Then:
     assertThat(errContent.toString(UTF_8),
-            containsString("Test failed: Failed to insert values into stream/table: TEST\n"));
+            containsString("Test failed: Failed to insert values into stream/table: " +
```

Comment:
I'd leave the test as it was, (which means the output needed to include the name of the target.